generate_workspace/src/main/java/com/google/devtools/build/workspace/output/AbstractWriter.java

```diff
@@ -25,14 +25,15 @@
 public abstract class AbstractWriter {
   public abstract void write(Collection<Rule> rules);
 
+  private static final String lineSeparator = System.lineSeparator();
```

Comment:
Is `System.lineSeparator)` equivalent to `String.format("%n")`?

Reply:
Yes!